server.js

```diff
@@ -533,3 +533,17 @@ setInterval(function() {
                 "Warning! All builders are idle, but there are " + queued + " PRs in the queue.");
     });
 }, THIRTY_MINUTES);
+
+setInterval(function() {
+    homu.retrieveSlaves(function(slaves) {
+		for(var slaveName in slaves){
+		    if(slaves[slaveName].hasOwnProperty("runningBuilds")){
+		        slaves[slaveName]["runningBuilds"].forEach(function(runningBuild){
+		            if(runningBuild.eta > .5){
+		                bot.say(config.channels[0], slaveName + " is past overdue! (ETA=" + runningBuild.eta + ")");
```

Comment:
We can make this output more useful by doing `moment.unix(slaves[slaveName]["runningBuilds"]["times"][0]).fromNow()` instead of the raw ETA value, which will provide us with a more readable string: https://momentjs.com/docs/#/displaying/fromnow/

This will require adding moment.js to the package.json and installing it locally, too.

Reply:
Thanks! I'll look into adding moment and implementing it here.